app/src/utils/misc.ts

```diff
@@ -25,13 +25,15 @@ export function chunkArray<T>(arr: T[], chunkSize: number): T[][] {
 const specialCommands = {
   $gw2_account$: /(^|\s)\w+\.\d{4}($|\s)/,
   $steam_friend$: /(^|\s)\d{8}($|\s)/,
-  $gw2_or_steam$: /(^|\s)\w+\.\d{4}($|\s)|(^|\s)\d{8}($|\s)/,
+  $gw2_or_steam$: /(^|\s)(\w+\.\d{4}|\d{8})($|\s)/,
+  $gw2_steam_paypal$: /(^|\s)(\w+\.\d{4}|\d{8}|paypal)($|\s)/
```

Comment:
[suggestion]

```suggestion
  $gw2_steam_paypal$: /(^|\s)(\w+\.\d{4}|\d{8}|paypal)($|\s)/i
```

Make the regex case insensitive so then `Paypal`/`paypal`/`PayPal` will all match. Same goes for the combination regex below.